test/unit/app/helpers/govspeak_helper_test.rb

```diff
@@ -197,6 +197,17 @@ def edition_with_attachment(body:)
     refute_select_within_html html, ".image.embedded"
   end
 
+  test "should ignore images with non-default image kinds" do
```

Comment:
It wouldn't be too much extra effort to add a `can_be_embedded_in_govspeak: true` property to the YAML config, so that we can set this at the config level rather than in code. Would also be less surprising for a dev following the breadcrumb trail when a support ticket comes in asking "why isn't my image rendering?" (explicit config as opposed to hardcoded conditional). What do you think?

Reply:
Done in 5f82aa5 - although I might be getting a bit too fancy with the method_missing stuff there - let me know if you think it's too much. I could go with a `permits?(string)` method instead, or just `permitted_uses.include?(string)` directly.

Reply:
Actually this is one of those where "if you have to ask 'is this too clever', it's probably too clever". I'll refactor it to use `permits?(use_case)`.

Reply:
Now done more simply in e0ade76